services/spar/test-integration/Spec.hs

```diff
@@ -39,8 +41,9 @@ partitionArgs = go [] []
 
 mkspec :: SpecWith TestEnv
 mkspec = do
-    describe "Test.Spar.API" Test.Spar.APISpec.spec
-    describe "Test.Spar.App" Test.Spar.AppSpec.spec
-    describe "Test.Spar.Data" Test.Spar.DataSpec.spec
-    describe "Test.Spar.Intra.Brig" Test.Spar.Intra.BrigSpec.spec
-    describe "Test.Spar.Scim" Test.Spar.ScimSpec.spec
+    describe "Spar.API" Test.Spar.APISpec.spec
+    describe "Spar.App" Test.Spar.AppSpec.spec
+    describe "Spar.Data" Test.Spar.DataSpec.spec
+    describe "Spar.Intra.Brig" Test.Spar.Intra.BrigSpec.spec
+    describe "Spar.Scim" Test.Spar.ScimSpec.spec
+    describe "Spar.Scim.Auth" Test.Spar.Scim.AuthSpec.spec
```

Comment:
I don't like this change.  `Test` was for the test cases, everything else for the tooling.  But ok, now it's changed.  I hope I won't change it back at some point.  :-)

Reply:
never mind, that comment made no sense.